03-how/01-new_members.html.md

```diff
@@ -6,16 +6,16 @@ title: New members FAQ
+* If you spot something that could bring value to Mangrove, check with others that it actually brings value, and do it. Or even, don’t check if you want and just do it, but then your contribution points / recognition in Mangrove linked to what you did might be subject to discussion. The only rule: if what you do impacts the work of others, talk to them before working on it. Just put in the effort and be creative about what could help the Mangrove lifestyle expand. \(\*\)
 
-* There is also an amount of day-to-day tasks to be done, but most of them require to have spent some time in mangrove \(pitches, answering emails..\) : new members are mostly contributing on projects
+* There are also some day-to-day tasks to be done, but most of them require to have spent some time in Mangrove \(pitches, answering emails..\) : new members mostly contribute through projects.
 
-The steps to achieve our vision are only ‘loosely’ planned, on purpose. We create a framework of emergence : in which the projects and next realisations emerge from the motivations of the members and the expressed needs from the ecosystem. What Mangrove will be in one year will emerge from all the energies we have inside, and how our frameworks canalise these energies toward common projects that matter. 'common projects', because to make the ecosystem grow in one direction - whatever this direction is, whichever is has been planned or not - some focus is needed. This is why the most important matters on the ecosystem will be upvoted and member taking part in the 2-weeks sprint teams to resolve them will be particularly rewarded.
+The steps to achieve our vision are only ‘loosely’ planned, on purpose. We create a framework of emergence : in which the projects and next realisations emerge from the motivations of the members and the expressed needs from the ecosystem. What Mangrove will be in one year will emerge from all the energy we have inside, and how our frameworks combine this energies towards common projects that matter. To make the ecosystem grow in one direction - whatever this direction is, whether it has been planned or not - some focus is needed. That is why we focus on this idea of "common projects that matter". The most important matters in the ecosystem will be voted on by the members. Members taking part in the 2-weeks sprint teams to resolve them will be particularly rewarded.
 
 _\(\*\) precision : we introduced a notion of rights \(with the 'tracks'\) to keep this freedom while taking more newcomers without endangering the integrity of the organisation : you cannot from the very beginning have the right to change the website only by yourself or integrate a new member. But you can help building peer to peer learning processes, work on co-living or co-working projects, work on the project helping people to switch to freelancing.._
```

Comment:
I had some trouble understanding this section and so wasn't able to improve it much.